client/src/pages/Attributes/Item/index.scss

```
@@ -0,0 +1,95 @@
+@use '~@/style/globals';
+
+.AttributesItem {
+    $block: &;
+
+    &__cell {
+        height: 45px;
+        padding: 0.8rem 0.5rem;
+        border-bottom: 1px solid globals.$bg-color-body;
+
+        &--empty {
+            color: globals.$text-light-color;
+            font-style: italic;
+        }
+
+        //
+        // - Specifics cells
```

```suggestion
        // - Specific cells
```

Mais sinon, on peut mettre les commentaires en FR non ? On l'a déjà fait ailleurs en tout cas... À moins qu'on change ? À voir.